package.json

```diff
@@ -18,16 +18,23 @@
   "dependencies": {
     "@heroicons/react": "^2.0.13",
     "@material-tailwind/react": "^1.2.4",
+    "@radix-ui/react-accordion": "^1.1.0",
```

Comment:
Does this include classnames? I'm getting this error when I pull and run the branch, even after `pnpm i`: 
![Screen Shot 2023-03-06 at 10 03 04 PM](https://user-images.githubusercontent.com/79157060/223309720-e32e239c-2270-44e2-8cf4-ac65fc711f97.png)

Reply:
updated `package.json` and `pnpm-lock.yaml`